CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
+### Changed
+
+- BC-Break: Changed - renamed namespace [#10](https://github.com/php-parallel-lint/PHP-Console-Color/pull/10) [@grogy](https://github.com/grogy).
```

Comment:
To make it more obvious what has changed, you may want to mention the "from" and the "to".

> The top-level namespace for all classes has changed from `JakubOnderka` to `PHP_Parallel_Lint`.

Reply:
You have right, thanks